sample/src/main/AndroidManifest.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.example.sample">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.infinum.libname.sample">
```

Comment:
I thought the package definition was thrown out of AndroidManifest 🤔 maybe it's the old Gradle version?

Reply:
I manually migrated so I forgot to remove it. Removing it now.